cijoe/tests/tools/test_xnvme.py

```diff
@@ -238,7 +238,36 @@ def test_dsm(cijoe, device, be_opts, cli_args):
     assert not err
 
 
-# For these three tests we request a device with labels 'ctrlr'
+# For these four tests we request a device with labels 'ctrlr'
```

Comment:
Perhaps this comment is redundant

Reply:
Maybe, but this stuff is "new" and a bit special, so, keeping it for now.